packages/iov-ethereum/src/ethereumconnection.ts

```diff
@@ -218,12 +218,29 @@ export class EthereumConnection implements BcpConnection {
   }
 
   public async getAllTickers(): Promise<ReadonlyArray<BcpTicker>> {
+    const erc20s = await Promise.all(
+      [...this.erc20Tokens.entries()].map(
+        async ([ticker, contract]): Promise<BcpTicker> => {
+          if (ticker !== (await contract.symbol())) {
+            throw new Error("Inconsistent ticker configured and in contract");
```

Comment:
Weird grammar. Suggest `Ticker ${ticker} does not match contract symbol ${symbol}`.

Reply:
Thanks for suggestion. I was not happy with the text as well but coder brain did not come up with proper language.